ansible/roles/discourse/defaults/main.yml

```diff
@@ -7,4 +7,4 @@ dis_guest_path: guest
 dis_path: /var/discourse
 dis_git: https://github.com/discourse/discourse_docker.git
 # Use a stable release - https://github.com/discourse/discourse/releases
-dis_version: v2.2.4
+dis_version: stable
```

Comment:
This should be using instead the version number of the stable version you want to deploy --usually the latest one.

If I make a guess from the date of the commit the stable version the server is currently using should be `v2.3.0`, however the current stable version is `v2.3.1` (was released 2 hours ago). It's of high importance to know the version of the services running in order to avoid conflicts, bugs and other horrible things that come with software. :)